lib/filesystem.nix

```diff
@@ -2,12 +2,94 @@
 { lib }:
```

Comment:
Could you remove/update the stale comment above this line, and perhaps tell us where the test is?
Something like:

```
# Tested by lib/tests/filesystem.sh
```

Reply:
Done with https://github.com/NixOS/nixpkgs/pull/224834/commits/378bf1a6192a5d46ceb7e07af7be1d37b93b47c7